Wrappers/Python/test/test_io.py

```diff
@@ -362,4 +365,87 @@ def TIFF_compression_test(self, compression, channels=1):
         np.testing.assert_array_equal(tmp, read_array)
 
         
+class TestRAW(unittest.TestCase):
+    def setUp(self) -> None:
+        # self.logger = logging.getLogger('cil.io')
+        # self.logger.setLevel(logging.DEBUG)
+        self.cwd = os.path.join(os.getcwd(), 'rawtest')
+        os.mkdir(self.cwd)
+
+    def tearDown(self) -> None:
+        shutil.rmtree(self.cwd)
+        # pass
+
+    def test_raw_nocompression_0(self):
+        self.RAW_compression_test(None,1)
+    
+    def test_raw_compression_0(self):
+        self.RAW_compression_test('uint8',1)
+
+    def test_raw_compression_1(self):
+        self.RAW_compression_test('uint16',1)
+
+    def test_raw_nocompression_1(self):
+        self.RAW_compression_test(None,1)
+    
+    def test_raw_compression_2(self):
+        with self.assertRaises(ValueError) as context:
+            self.RAW_compression_test(12,1)
+
+    def RAW_compression_test(self, compression, channels=1):
+        X=4
+        Y=5
+        Z=6
+        C=channels
+        if C == 1:
+            ig = ImageGeometry(voxel_num_x=4, voxel_num_y=5, voxel_num_z=6)
+        else:
+            ig = ImageGeometry(voxel_num_x=4, voxel_num_y=5, voxel_num_z=6, channels=C)
+        data = ig.allocate(0)
+        data.fill(np.arange(X*Y*Z*C).reshape(ig.shape))
+
+        compress = utilities.get_compress(compression)
+        dtype = utilities.get_compressed_dtype(data.array, compression)
+        scale, offset = utilities.get_compression_scale_offset(data.array, compression)
+        if C > 1:
+            assert data.ndim == 4
+        fname = os.path.join(self.cwd, "unittest.raw")
+        
+        writer = RAWFileWriter(data=data, file_name=fname, compression=compression)
+        writer.write()
+
+        # read the data from the ini file
+        config = configparser.ConfigParser()
+        inifname = os.path.join(self.cwd, "unittest.ini")
+        config.read(inifname)
+
+        # read how to read the data from the ini file
+        read_dtype = config['MINIMAL INFO']['data_type']
+        read_array = np.fromfile(fname, dtype=read_dtype)
+        read_shape = eval(config['MINIMAL INFO']['shape'])
+        logging.warning("read_array shape {}".format(read_array.shape))
+        logging.warning("read_array dtype {}".format(read_array.dtype))
+
+        # reshape read in array
+        read_array = read_array.reshape(read_shape)
```

Comment:
is this needed?
You don't do this in the example docstring of reading data from the '.ini' file